stencil-library/src/styles/root.scss

```diff
@@ -30,6 +30,16 @@
   --jfi-form-control-margin: 0;
   --jfi-form-control-padding: #{$input-padding-y $input-padding-x};
 
+  // form radio group
+  --jfi-form-control-radio-button-color: #{$primary};
```

Comment:
Could we just have one set of new variables called `--jfi-button-{property}`?  To bootstrap, both are buttons and we should be able to apply the variables to both the button group and submit button

Reply:
The buttons within groups and outside of them need different styling, since their default state (i.e. when not "checked" in the case of button groups) don't match.

Reply:
What we could do is add some variables that are used in other variables to easily override many at the same time, like:
`--jfi-primary: {#primary}`. Then that can be used in many places at the same time so you can override them separately, but also just change the primary color and change them all at once.